__tests__/fixtures/install/install-production-without-dev/yarn.lock

```diff
@@ -0,0 +1,1219 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Sure. I'll try to come up with something else. Ensure that is fails pre-fix and ensure it is green post-fix.